app/src/util/networks.ts

```diff
@@ -260,13 +260,66 @@ export const getInfuraUrl = (networkId: number): string => {
 export const knownTokens: { [name in KnownToken]: KnownTokenData } = {
   cdai: {
     symbol: 'cDAI',
-    decimals: 18,
```

Comment:
![Screenshot 2021-01-20 at 23 31 04](https://user-images.githubusercontent.com/33226956/105249000-a9075d00-5b77-11eb-8564-7b272c9ce663.png)
Would be good to have consistent letter formatting for both currencies. To make them both capital or with first letter Uppercase (which is according to design...

Reply:
Yup, makes sense refactored to include the change.